dbOperations.php

```diff
@@ -63,6 +88,169 @@ public function delete($employeeId) {
 
 	}
 
+	/**
+     *Selects all the details of all the employees
+     *
+     * @access public
+     * @param void 
+     * @return object/boolean
+     */
+	public function selectAllEmployees() {
+		
+        //Get the records of all the employees
+        $selectEmpDetails = "SELECT employee.eid, employee.firstName, employee.middleName,
```

Comment:
Can make this query string more readable by introducing line breaks currently it looks like a paragraph.